packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -277,7 +273,10 @@ export const prepareStatusManager = (
       },
 
       /**
-       * Remove and return an `ADVANCED` or `OBSERVED` tx waiting to be `SETTLED`.
+       * Remove and return the _first_ `PendingSettleTx` that matches the observed
+       * settlement. This is not necessarily an exact match with the original
```

Comment:
what does "not necessarily an exact match" mean? isn't "same forwarding account and amount" the relevant definition of "matching"?

Reply:
The ordering could be different if Alice requests two advances for the same amount.

Reply:
It is an exact match based on available data, which treats (EUD,Amount) interchangeably. 

I think think what you're getting at is that a mint deposit may be for any of the matching transactions and we can't distinguish between them, so we go with the first.

Reply:
How's this:

```js
/**
 * Remove and return the oldest pending settlement transaction that matches the given
 * forwarding account and amount. Since multiple pending transactions may exist with
 * identical (account, amount) pairs, we process them in FIFO order.
 */
```